### PR TITLE
sci-libs/opencascade: Version bump to 7.3.0

### PR DESCRIPTION
**Pull request by Perlovka**
Closes: https://bugs.gentoo.org/610362
Package-Manager: Portage-2.3.49, Repoman-2.3.10

Based on waebbl ebuild (https://github.com/waebbl/waebbl-gentoo/tree/master/sci-libs/opencascade)
Reworked and simplified a bit.
FreeCAD 0.18 git master been succesfully built with this version and have no issues so far. (Crashed with opencascade-6.9.1 on BSpline operations)

### CODE REVIEW

**Comment by a17r** [comment]
Please check out bug https://bugs.gentoo.org/665430

File: sci-libs/opencascade/opencascade-7.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+		-DUSE_VTK=$(usex vtk)
+	)
+
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	local my_libdir="$(get_libdir)"
```
Comment (Zlogene) [inline sci-libs/opencascade/opencascade-7.3.0.ebuild]: have no clue why you introduce garbage variables, it is the same workload

File: sci-libs/opencascade/opencascade-7.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+src_prepare() {
+	cmake-utils_src_prepare
+	use java && java-pkg-opt-2_src_prepare
+
+	# Do not pre-strip files
+	sed -i 's/_FLAGS_RELEASE} -s/_FLAGS_RELEASE}/g' adm/cmake/occt_defs_flags.cmake || die
+
+	# Prepare environment variables used by Opencascade
+	local my_install_dir="${EROOT}usr/$(get_libdir)/${P}"
```
Comment (Zlogene) [inline sci-libs/opencascade/opencascade-7.3.0.ebuild]: do not introdice unneeded statements

**Comment by Perlovka** [comment]
@a17r This is slotted version. Should not be a problem. Opencascase 7.2.0 mentioned in a bug seems to be unslotted one.

**Comment by Perlovka** [comment]
I have this version in my overlay, if someone willing to test compatibility with other programs.
https://github.com/Perlovka/portage-overlay

**Comment by Perlovka** [comment]
sci-libs/oce has benn built successfully along with opencascade.

@Zlogene thanks for the review, I'll fix this later, have to go.

File: sci-libs/opencascade/opencascade-7.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit check-reqs cmake-utils eapi7-ver java-pkg-opt-2 multilib
```
[the remark this answers is not in the document]
Comment (Perlovka) [reply sci-libs/opencascade/opencascade-7.3.0.ebuild]: Umm. There are no eutils )

[the remark this answers is not in the document]
Comment (Zlogene) [reply sci-libs/opencascade/opencascade-7.3.0.ebuild]: @Perlovka sigh, I meant `multilib`

**Comment by Perlovka** [comment]
Fixed.

**Comment by Perlovka** [comment]
Sorted dependencies and rebased.

**Comment by Perlovka** [comment]
@Zlogene have you received my answers to comments? Seems rebasing deletes comments from code.

**Comment by Perlovka** [comment]
Fixed vtk dependency, must be build with rendering USE flag.

**Comment by monsieurp** [comment]
@Zlogene: ping.

**Comment by a17r** [comment]
Please add your sign-off, then we can merge this per maintainer timeout.

**Pull request closed without merging**